docs/source/raft_ann_benchmarks.md

```diff
@@ -347,6 +222,149 @@ The figure below is the resulting plot of running our benchmarks as of August 20
+    "--algorithms raft_cagra,raft_ivf_pq --batch-size 10 -k 10" \
+    ""
+```
+
+Usage of the above command is as follows:
+
+| Argument                                                  | Description                                                                                        |
+|-----------------------------------------------------------|----------------------------------------------------------------------------------------------------|
+| `rapidsai/raft-ann-bench:23.12a-cuda11.8-py3.10`          | Image to use. Can be either `raft-ann-bench` or `raft-ann-bench-datasets`                          |
+| `"--dataset deep-image-96-angular"`                       | Dataset name                                                                                       |
+| `"--normalize"`                                           | Whether to normalize the dataset                                                                   |
+| `"--algorithms raft_cagra,hnswlib --batch-size 10 -k 10"` | Arguments passed to the `run` script, such as the algorithms to benchmark, the batch size, and `k` |
+| `""`                                                      | Additional (optional) arguments that will be passed to the `plot` script.                          |
+
+***Note about user and file permissions:*** The flag `-u $(id -u)` allows the user inside the container to match the `uid` of the user outside the container, allowing the container to read and write to the mounted volume indicated by the `$DATA_FOLDER` variable.
+
+#### End-to-end run on CPU
+
+The container arguments in the above section also be used for the CPU-only container, which can be used on systems that don't have a GPU installed. 
+
+Note the image changes to `raft-ann-bench-cpu` container and the `--gpus all` argument is no longer used:
+```bash
+export DATA_FOLDER=path/to/store/datasets/and/results
+docker run  --rm -it -u $(id -u)                  \
+    -v $DATA_FOLDER:/data/benchmarks              \
+    rapidsai/raft-ann-bench-cpu:23.12a-py3.10     \
+     "--dataset deep-image-96-angular"            \
+     "--normalize"                                \
+     "--algorithms hnswlib --batch-size 10 -k 10" \
+     ""
+```
+
+#### Manually run the scripts inside the container
+
+All of the `raft-ann-bench` images contain the Conda packages, so they can be used directly by logging directly into the container itself:
+
+```bash
+export DATA_FOLDER=path/to/store/datasets/and/results
+docker run --gpus all --rm -it -u $(id -u)          \
+    --entrypoint /bin/bash                          \
+    --workdir /data/benchmarks                      \
+    -v $DATA_FOLDER:/data/benchmarks                \
+    rapidsai/raft-ann-bench:23.12a-cuda11.8-py3.10 
+```
+
+This will drop you into a command line in the container, with the `raft-ann-bench` python package ready to use, as described in the [Running the benchmarks](#running-the-benchmarks) above:
```

Comment:
```suggestion
This will drop you into a command line in the container, with the `raft-ann-bench` python package ready to use, as described in the [Running the benchmarks](#running-the-benchmarks) section above:
```